docs/_includes/calendar.html

```diff
@@ -274,14 +274,14 @@ <h3>Calendar with Header</h3>
                 <div class="calendar__header">
                     <div class="calendar__header--inner">
                         <button variant="icon" class="icon-btn icon-btn--small icon-btn--transparent" aria-label="Previous Month - July 2019">
-                            <svg aria-hidden="true" class="icon icon--chevron-left-24" focusable="false">
-                                {% include symbol.html name="chevron-left-24" %}
+                            <svg aria-hidden="true" class="icon icon--chevron-left" focusable="false">
```

Comment:
A patch has been released in `15.5.1` to fix the icon issue. So we can switch this back to use the new icon.

Reply:
Lets just get this out for now. I will take care of the icons part when i switch all the icons in skin to the new ones.

Reply:
Ok.